src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -14,6 +14,11 @@
             'sfwdt{{ token }}',
             '{{ path("_wdt", { "token": token }) }}',
             function(xhr, el) {
+
+                /* Evaluate embedded scripts inside the toolbar */
+                var rx=/<script>(.*?)<\/script>/g, s;
+                while (s = rx.exec(el.innerHTML)) {eval(s[1]);};
```

Comment:
do we actually need this ? It would force to enable eval in the CSP, which looks like a big change compared to the CSP defined in the project (and which could lead to CSP violations in prod due to using eval somewhere while the local setup was fine with it due to the debug tools changing it)